stonesoup/hypothesiser/distance.py

```diff
@@ -66,21 +65,23 @@ def hypothesise(self, track, detections, timestamp):
         hypotheses = list()
 
         # Common state & measurement prediction
-        prediction = self.predictor.predict(track.state, timestamp=timestamp)
-
+        prediction = self.predictor.predict(track, timestamp=timestamp)
+        measurement_prediction = self.updater.predict_measurement(
+            prediction)
         # Missed detection hypothesis with distance as 'missed_distance'
         hypotheses.append(
             SingleDistanceHypothesis(
                 prediction,
                 MissedDetection(timestamp=timestamp),
-                self.missed_distance))
+                self.missed_distance,
+                measurement_prediction))
```

Comment:
Does not having a `measurement_prediction` here cause an issue?

Reply:
`measurement_prediction` is on L69 or do you mean since there isn't a measurement here?